kmm/kmm-feed/src/commonMain/kotlin/com/gchristov/newsfeed/kmmfeed/FeedViewModel.kt

```diff
@@ -134,7 +134,7 @@ class FeedViewModel(
         val blockingError: Throwable? = null,
         val nonBlockingError: Throwable? = null,
         val sectionedFeed: SectionedFeed? = null,
-        val searchQuery: String? = null
+        val searchQuery: String = "brexit,fintech"
```

Comment:
Ah this is already made non null. Then we don’t have to handle in the UI and you can ignore my prev comment on that